$NetBSD: patch-aa,v 1.15 2012/03/02 15:24:48 wiz Exp $

Use pkgsrc's COMPILER_RPATH_FLAG variable, so this works on all pkgsrc platforms.

--- pcre-config.in.orig	2012-01-21 16:00:28.000000000 +0000
+++ pcre-config.in
@@ -28,15 +28,7 @@ if test $# -eq 0; then
       exit 1
 fi
 
-libR=
-case `uname -s` in
-  *SunOS*)
-  libR=" -R@libdir@"
-  ;;
-  *BSD*)
-  libR=" -Wl,-R@libdir@"
-  ;;
-esac
+libR=" @COMPILER_RPATH_FLAG@@libdir@"
 
 libS=
 if test @libdir@ != /usr/lib ; then
